src/components/sections/judge/JudgeRewardsPayoutTableRow.tsx

```diff
@@ -24,14 +26,28 @@ const JudgeRewardsPayoutTableRow = ({
     })
   }
 
+  const copyWalletAddress = async () => {
+    if (!reward.user?.walletAddress) return
+    await navigator.clipboard.writeText(reward.user.walletAddress)
+    toast({
+      title: 'Copied to Clipboard',
+      description: `Wallet address ${formatTxHash(reward.user.walletAddress)} copied to clipboard`,
+    })
+  }
+
   return (
     <TableRow>
       <TableCell className="text-titleS">
         {reward.user?.name ?? reward.user?.alias ?? '-'}
       </TableCell>
       <TableCell className="text-bodyM">{reward.user?.email ?? '-'}</TableCell>
       <TableCell className="text-bodyM">
-        {reward.user?.walletAddress ?? '-'}
+        <div className="flex items-center gap-2">
+          {formatTxHash(reward.user?.walletAddress)}
```

Comment:
This can be a bit confusing to use `formatTxHash` on wallet address. It would be better to have standalone `formatAddress` function, if you don't want to do any special address formatting, for now it can be set to 
```
const formatAddress = formatTxHash;
```
but you might also want to consider setting some more sensible defaults for addresses, for example the `addr_` part is always the same, so there could be more characters visible at the beggining.